.github/workflows/on-remove.yml

```diff
@@ -214,15 +214,15 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-default-region: ${{ steps.setup.outputs.aws-default-region }}
       - name: Delete the entity index
-        if: ${{ steps.setup.outputs.crn-contentful-on-branch-env == 'true' && (steps.setup.outputs.crn-algolia-index != 'asap-hub_dev' || steps.setup.outputs.crn-algolia-index != 'asap-hub_prod')}}
```

Comment:
I believe here we should have `&&` instead of `||` because, for example, if we have `crn-algolia-index` = `'asap-hub_dev'`

`steps.setup.outputs.crn-algolia-index != 'asap-hub_dev'` would be `false`
`steps.setup.outputs.crn-algolia-index != 'asap-hub_prod'` would be `true`

so `(steps.setup.outputs.crn-algolia-index != 'asap-hub_dev' || steps.setup.outputs.crn-algolia-index != 'asap-hub_prod')` would be `false || true` -> `true` and it could delete `asap-hub_dev` eventually

Reply:
To be fair it should never be `crn-contentful-on-branch-env` and `crn-algolia-index`.

Reply:
yeah, but I think it's safer to leave this condition here 😅